src/endpoints/cart.js

```diff
@@ -67,12 +67,14 @@ class CartEndpoint extends BaseExtend {
     )
   }
 
-  BulkAdd(body, options) {
-    const payload = options ? { data: body, options } : body
+  BulkAdd(body) {
     return this.request.send(
       `${this.endpoint}/${this.cartId}/items`,
       'POST',
-      payload
+      {
+        data: body,
+        options: { add_all_or_nothing: false}
```

Comment:
Linter failing this check in build. Please add a space after the last curly brace
![image](https://user-images.githubusercontent.com/39800563/103550776-66573c80-4e77-11eb-8580-56602d977bcc.png)